mod_cluster/extension/src/test/java/org/wildfly/extension/mod_cluster/ModClusterSubsystemTestCase.java

```diff
@@ -410,7 +427,7 @@ public void testSSL() throws Exception {
         Assert.assertEquals("SSL_DHE_DSS_WITH_3DES_EDE_CBC_SHA,SSL_RSA_WITH_RC4_128_MD5,SSL_RSA_WITH_RC4_128_SHA,SSL_RSA_WITH_3DES_EDE_CBC_SHA", ssl.get("cipher-suite").resolve().asString());
         Assert.assertEquals("mykeyalias", ssl.get("key-alias").resolve().asString());
         Assert.assertEquals("mypassword", ssl.get("password").resolve().asString());
-        Assert.assertEquals("TLS", ssl.get("protocol").resolve().asString());
```

Comment:
My feelings are that TLS is probably a wrong value, the enumeration must be checked to included only safe protocols (no SSLv2 nor SSLv3).